inference-engine/tests/functional/inference_engine/transformations/depth_to_space_fusion_test.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/test_common.hpp"
+#include <string>
+#include <sstream>
+#include <fstream>
+#include <memory>
+#include <queue>
+#include <map>
+
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/pass/constant_folding.hpp>
+#include <ngraph_ops/fully_connected.hpp>
+#include <transformations/depth_to_space_fusion.hpp>
+#include <transformations/utils/utils.hpp>
+#include <transformations/init_node_info.hpp>
+
+#include "ngraph_test_utils.hpp"
+
+using namespace testing;
+
+TEST(TransformationTests, DepthToSpaceFusionDepthFirst) {
```

Comment:
Please add negative cases and cases with dynamic shapes.

Reply:
You still don't have test cases with dynamic shapes.
You can use PartialShape{DYN, 1, 2, 3} to specify dynamic shapes.

Reply:
DepthToSpaceFusionDynamicShape test: auto shape_reshape_before = std::make_shared<ngraph::opset3::Parameter>(ngraph::element::i64, ngraph::Shape{6});

Reply:
The next reshape op will have dynamic shapes after infer function

Reply:
DepthToSpaceFusionDynamicShape test: auto shape_reshape_before = std::make_sharedngraph::opset3::Parameter(ngraph::element::i64, ngraph::Shape{6}); - produces static shape.